src/lib/structures/Josh.ts

```diff
@@ -368,6 +368,27 @@ export class Josh<StoredValue = unknown> {
     return this;
   }
 
+  /**
+   * Loop over every key-value pair in this {@link Josh} and execute `hook` on it.
+   * @param hook The hook function to execute with each key.
+   */
+  public async each(hook: (value: any, key: string) => void): Promise<this> {
```

Comment:
`any` is a no go, please reference the other hooks used in this file and how they are referenced.

Reply:
*whoops, oversight!*